ReleaseProcessAutomation.UnitTests/SemanticVersioning/SemanticVersionTests.cs

```diff
@@ -173,6 +252,73 @@ public void GetCurrentPossibleVersionsHotfix_WithPreReleaseRC_ReturnsVersions ()
 
     Assert.That(possibleVersions, Is.EqualTo(expectedVersions));
   }
+  
+  [Test]
+  public void GetNextPossibleVersionsReleaseBranchFromDevelop_WithPreReleaseRC_ReturnsExpectedVersions ()
+  {
+    var parser = new SemanticVersionParser();
+    var expectedVersions = new SemanticVersion[]
+                           {
+                               parser.ParseVersion("1.3.5-rc.2"),
+                               parser.ParseVersion("1.3.5")
+                           };
+    var baseVersion = parser.ParseVersion("1.3.5-rc.1");
+
+    var possibleVersions = baseVersion.GetNextPossibleVersionsForReleaseBranchFromDevelop();
+
+    Assert.That(possibleVersions, Is.EqualTo(expectedVersions));
+  }
+  
+  [Test]
+  public void GetNextPossibleVersionsReleaseBranchFromDevelop_WithFullRelease_ReturnsExpectedVersions ()
+  {
+    var parser = new SemanticVersionParser();
+    var expectedVersions = new SemanticVersion[]
+                           {                               
+                               parser.ParseVersion("2.1.0"),
```

Comment:
should we add 2.1.0-alpha and beta, too?

and a separate topic (ticket): we should also add logic, that the associated release-version is added implicitly to jira, when a prerelease version is selected. i.e. it's not helpful to decide we want a 2.1.0-alpha.1 when there's not also a 2.1.0 version in jira.

Reply:
Created https://re-motion.atlassian.net/browse/RMRPA-11